iaso/tests/api/test_setup_account.py

```
@@ -163,3 +170,19 @@ def test_setup_account_project_creation(self):
         data_source = created_data_source.first()
         project_data_sources = project.data_sources.filter(pk=data_source.id)
         self.assertEqual(project_data_sources.first(), data_source)
+
+    def test_setup_account_with_feature_flags(self):
```

I would add more tests:
- One that should raise an exception with an invalid value in feature flags
- One that doesn't send any feature flag (to see if the default 5 flags are there)

---

I've added 2 additional tests:
- one with at least 1 invalid value in the feature flags
- another one without feature flags.